src/main/kotlin/net/leanix/vsm/sbomBooster/service/ProcessService.kt

```diff
@@ -61,4 +65,11 @@ class ProcessService(
             "Processed repository with url: $projectUrl in $duration"
         )
     }
+
+    private fun extractVsmRegion(accessToken: String): String {
+        val decoder = Base64.getDecoder()
+        val payload = String(decoder.decode(accessToken.split(".")[1]))
+
+        return payload.substringAfter("\"iss\":\"https://").substringBefore("-svc.leanix.net")
```

Comment:
I don't think this works. We give specific domains to customers like:
https://adidas.leanix.net

Reply:
we checked, this works. @geoandri has the details. we also use it in the [GH action ](https://github.com/leanix/vsm-discovery-github-action/blob/77f97b8b993e36a568ae294e3001420c67da317c/src/mtm.js#L4). maybe worth to share in the team, @geoandri

Reply:
We do have the same logic in our vsm discovery github action (https://github.com/leanix/vsm-discovery-github-action/blob/main/src/mtm.js). 
To my understanding, the `iss` property always has the region. I saw in the tokens that there is a property `instanceUrl` that has the custom domain. 

For example, for a token from the leanix vsm workspace I have 

`"iss": "https://eu-svc.leanix.net",` and `"instanceUrl": "https://leanix.leanix.net"`